proto/pdpb.proto

```diff
@@ -103,33 +103,43 @@ service PD {
 }
 
 message WatchGlobalConfigRequest {
-
+    string config_path = 1;
+    int64 revision = 2;
 }
 
 message WatchGlobalConfigResponse {
     repeated GlobalConfigItem changes = 1;
+    int64 revision = 2;
 }
 
 message StoreGlobalConfigRequest {
-    repeated GlobalConfigItem changes = 1;
+    string config_path = 1;
+    repeated GlobalConfigItem changes = 2;
 }
 
 message StoreGlobalConfigResponse {
     Error error = 1;
 }
 
 message LoadGlobalConfigRequest {
-    repeated string names = 1;
+    string config_path = 1;
 }
 
 message LoadGlobalConfigResponse {
     repeated GlobalConfigItem items = 1;
+    int64 revision = 2;
+}
+
+enum ItemKind {
```

Comment:
How about calling it EventType or something else which can represent that it is an action?

Reply:
You are right! from that way we can sync with Etcd